test_all.py

```diff
@@ -167,6 +167,20 @@ def test_model_onnx_detection(self):
             tolerate_small_mismatch=True,
         )
 
+    def test_model_onnx_detection_panoptic(self):
```

Comment:
Can you add a `unittest.skip("CI doesn't have enough memory")` so that we can merge this PR?